### DISCUSSION TITLE
add global search  function in vnote

### DISCUSSION THREAD

**Discussion by zcjjfish** [tamlok/vnote#2519] [General]
希望能增加在vnote book 里面增加一个全局搜索功能哦，谢谢

**Comment by tamlok** [comment]
已经支持了：

![Uploading image.png…]()

或者按Ctrl+G, G快捷键

**Comment by zcjjfish** [comment]
给力！



---Original---
From: "Le ***@***.***&gt;
Date: Thu, Jun 27, 2024 20:15 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [vnotex/vnote] add global search  function in vnote (Discussion#2519)




 
已经支持了：
 

 
或者按Ctrl+G, G快捷键
